moon-store-mac-os/moon-store-mac-os/Services/BaseNetworkService.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  BaseNetworkService.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/3/24.
+//
+
+import Foundation
+
```

Comment:
Es mejor agregar directamente el codigo de error, al menos el enum. Asi ya se ve la capa de networking bien, ya que hay muchas cosas que se mencionan que necesita esto @Rexmoon

Reply:
On it!